pkg/graph/schema.graphql

```diff
@@ -1303,10 +1306,14 @@ type OperationalSolution {
 
 input OperationalSolutionChanges @goModel(model: "map[string]interface{}"){
     needed: Boolean
+    name: String

```

Comment:
Can you remind me when `name` needs to get passed? (Perhaps it could use a comment like the one for `nameOther`)

Reply:
Good call Clay! I think you are right, that this shouldn't be in the changes object. Name should be coming from possible operational solution, I will update this!